### DISCUSSION TITLE
NBT Editing + Client Chest Access In Survival

### DISCUSSION THREAD

**Discussion by HyperlightDrifta** [mega12345mega/NBT-Editor#84] [Ideas]
In World Options/Mod Config/Server Config If the Mod Is Being Used On a Server. You Can Allow Using NBT Editor and/or The Client Chest in Survival/Adventure Mode.

**Comment by mega12345mega** [comment]
I think this is covered by #83. I almost have a dev build ready, but there are a couple of bugs I want to fix first.

**Reply by HyperlightDrifta** [reply]
Awesome! I was looking for something like what I mentioned in this post so I'm really happy to see that my dreams of this mod are coming to life.

**Comment by HyperlightDrifta** [comment]
*edited typo in original discussion

**Comment by HyperlightDrifta** [comment]
Is /get command allowed in survival/adventure included as well?

**Reply by mega12345mega** [reply]
Yah, all of the item stuff uses the same packet to set a slot to an item, so it should all work.
Here's the dev build BTW: [https://github.com/mega12345mega/NBT-Editor/actions/runs/7779836726](https://github.com/mega12345mega/NBT-Editor/actions/runs/7779836726); download the `Artifacts.zip` and use `libs/nbteditor-2.0.0.999.jar`. It hasn't been tested extensively outside of the development environment, but it works in theory.

**Reply by HyperlightDrifta** [reply]
Thanks. I swear this'll be me last question, but how exactly do you allow it to be used in survival?

**Reply by mega12345mega** [reply]
It should be enabled automatically as long as the mod is installed on the server (or you're in singleplayer) and you have OP.
